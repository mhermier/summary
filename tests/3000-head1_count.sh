#!/bin/sh

utils/generate-test-input.sh 0 100 0 | ./summary --head 1 --count --count-skipped --count-matched
