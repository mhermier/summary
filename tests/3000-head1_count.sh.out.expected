entry-1
... 99 entries skipped
1/100 entries matched
